example/index.html

```diff
@@ -9,8 +9,8 @@
             <li><a href="/screenOff">Screen Off</a></li>
             <li><a href="/isScreenOn">Get screen state</a></li>
             <li><a href="/getBrightness">Get actual Brightness</a></li>
-            <li><a href="/setBrightness/50">Set Brigthness value to 50</a></li>
-            <li><a href="/getMaxBrightness">Get Max Brightness</a></li>
+            <li><a href="/setBrightness/10">Set Brigthness value to 10%</a></li>
+            <li><a href="/setBrightness/85/45">Set Brigthness value to 85% with a light smooth effect</a></li>
```

Comment:
Sorry, this was preexisting but I missed it before: `Brigthness` should be `Brightness`